src/main/java/seedu/address/logic/parser/reminder/AddCommandParser.java

```diff
@@ -0,0 +1,53 @@
+package seedu.address.logic.parser.reminder;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_CONTACT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_DATETIME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_MESSAGE;
+import static seedu.address.logic.parser.ParserUtil.arePrefixesPresent;
+
+import java.time.LocalDateTime;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.reminder.AddCommand;
+import seedu.address.logic.parser.ArgumentMultimap;
+import seedu.address.logic.parser.ArgumentTokenizer;
+import seedu.address.logic.parser.Parser;
+import seedu.address.logic.parser.ParserUtil;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new AddCommand object for Reminder
+ */
+public class AddCommandParser implements Parser<AddCommand> {
+
+    /**
+     * Parses the given {@code String} of arguments in the context of the AddCommand
+     * and returns an AddCommand object for execution.
+     *
+     * @throws ParseException if the user input does not conform to the expected format
```

Comment:
Should there be a full stop at the end of the javadoc? Very minor thing though! Is also in some other files